cli/formatter/formatter.go

```diff
@@ -0,0 +1,778 @@
+package formatter
+
+import (
+	"encoding/json"
+	"fmt"
+	"reflect"
+	"sort"
+	"strconv"
+	"strings"
+	"unicode/utf8"
+
+	"github.com/apex/log"
+	"github.com/tarantool/tt/cli/formatter/table"
+	"github.com/tarantool/tt/cli/formatter/text"
+	"gopkg.in/yaml.v2"
+)
+
+// Opts contains formatting options.
+type Opts struct {
+	TransposeMode bool
+	NoGraphics    bool
+	ColWidthMax   int
+	TableDialect  TableDialect
+}
+
+// TableDialect defines a set of supported table dialect.
+type TableDialect int
```

Comment:
It's confusing to see `OutputFormat` in the `connect/output.go`, but `TableDialect` at `formatter/formatter.go` strange to see here. I see it as abstractions of the same level and the same responsibility. So I suggest to move `OutputFormat` to `formatter/format.go`.

Reply:
Done